scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -308,6 +379,7 @@ object Defn {
       mods: List[Mod],
       name: scala.meta.Type.Name,
       tparams: List[scala.meta.Type.Param],
+      bounds: scala.meta.Type.Bounds,
```

Comment:
This is a binary breaking change, can you elaborate on the use-case here? If this is Dotty-only syntax then I propose we introduce a new class instead so that we don't break existing usage of `Defn.Type`

Reply:
needed for opaque type aliases `opaque type A <: B = C`

https://dotty.epfl.ch/docs/reference/other-new-features/opaques.html

Yes it is a new syntax, I will introduce a new type.

Reply:
created completely new type `OpaqueTypeAlias`